web_widget_many2many_tags_multi_selection/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Tags multiple selection',
+    'version': '11.0.1.0.0',
+    'author': 'Akretion, Odoo Community Association (OCA), Jamin Shah',
+    'depends': [
+        'web',
+    ],
+    'demo': [],
```

Comment:
It's not really important to me but it is in the contributors guide line https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md , please remove empty keys